bin/scons_dev_master.py

```diff
@@ -152,11 +152,11 @@ def main(argv=None):
                 sys.exit(0)
             elif o in ('-n', '--no-exec'):
                 CommandRunner.execute = CommandRunner.do_not_execute
-            elif o in ('--password'):
+            elif o in '--password':
```

Comment:
Shouldn't these just be == instead of in?

Reply:
the suggestion (from pycharm) was just to remove them; to me it later looks like it was trying to keep a consistent model like  `elif o in (tuple of options)` and so I changed some back to a tuple but guess I missed this one... was intending `elif o in ('--password',)`.  changing it to equality is fine as well, no biggy to me.